anaconda_project/project.py

```diff
@@ -842,6 +844,10 @@ def _update_conda_env_requirements(self, requirements, problems, project_file):
         if _fatal_problem(problems):
             return
 
+        if 'bootstrap-env' in self.env_specs:
```

Comment:
This check feels a but manual, and is repeated on several places, maybe we could have a helper method?

Why not use `def has_bootstrap_env_spec(self):` ?

Reply:
right.. especially since it is already there.. 😄